### PR TITLE
optimize: added interface to get current transaction role

### LINKED ISSUE #3532: AOP动态创建/关闭Seata分布式事务中怎么获取GlobalTransactionRole
seata版本1.4.0
开发环境为 idea 
注册中心eureka
AOP实现的demo：(http://seata.io/zh-cn/blog/seata-spring-boot-aop-aspectj.html)
_________________________________________________
 RT：
     AOP中通过@Before来发起或者参与事务，参考源码GlobalTransactionContext中的
  [ public static GlobalTransaction getCurrentOrCreate() {
        GlobalTransaction tx = getCurrent();
        if (tx == null) {
            return createNew();
        }
        return tx;
    }](url)

当@AfterReturning时API方式中没有获取当前 GlobalTransactionRole的方法？？？

### PR DESCRIPTION
**Pull request by a364176773**
### Ⅱ. Does this pull request fix one issue?

fixes #3532

### CODE REVIEW

**Review by slievrly — APPROVED** [review]
LGTM

File: changes/en-us/1.5.0.md [hunk 1]
```diff
@@ -65,6 +65,7 @@
   - [[#3415](https://github.com/seata/seata/pull/3415)] optimize maven clean plugin to clear the distribution directory 	
   - [[#3316](https://github.com/seata/seata/pull/3316)] optimize the property bean may not be initialized while reading config value	
   - [[#3420](https://github.com/seata/seata/pull/3420)] optimize enumerated classes and add unit tests	
+  - [[#3436](https://github.com/seata/seata/pull/3436)] added interface to get current transaction role
```
Comment (caohdgege) [inline changes/en-us/1.5.0.md:68]: 3533

Comment (a364176773) [reply changes/en-us/1.5.0.md]: done

**Review by l81893521 — APPROVED** [review]
LGTM

**Pull request merged**